buildkite/benchmark/run.py

```diff
@@ -50,6 +53,53 @@
             "ARROW_SRC": f"{build_dir}/arrow",  # required by Java Script benchmarks
         },
     },
+    {
+        "benchmarkable_type": "arrow-commit",
+        "repo": "https://github.com/voltrondata-labs/arrowbench.git",
+        "root": "arrowbench",
+        "branch": "main",
+        "setup_commands": [],
+        "path_to_benchmark_groups_list_json": "arrowbench/inst/benchmarks.json",
+        "url_for_benchmark_groups_list_json": "https://raw.githubusercontent.com/voltrondata-labs/arrowbench/main/inst/benchmarks.json",
+        "setup_commands_for_lang_benchmarks": {  # These commands need to be defined as functions in buildkite/benchmark/utils.sh
+            "R": [
+                "build_arrow_r",
+                "install_arrowbench",
+                "create_data_dir",
+            ],
+        },
+        "env_vars": {
+            "PYTHONFAULTHANDLER": "1",  # makes it easy to debug segmentation faults
+            "BENCHMARKS_DATA_DIR": f"{os.getenv('HOME')}/data",  # allows to avoid loading Python and R benchmarks input data from s3 for every build
+            "ARROWBENCH_DATA_DIR": f"{os.getenv('HOME')}/data",  # allows to avoid loading R benchmarks input data from s3 for every build
```

Comment:
If this is for arrowbench we only need one of these (`ARROWBENCH_DATA_DIR`), right?

Reply:
Yep, removing